### PR TITLE
feat: add new subpage

### PR DESCRIPTION
**Pull request by Ervige**

### CODE REVIEW

File: pages/housePrideQuiz/hufflepuff/hufflepuffNewt.html [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+                Privacy & Cookies
+              </a>
+            </li>
+            <li class="mobileFooter__item">
+              <a
+                href="#"
+                class="mobileFooter__link"
+                target="_blank"
+                rel="noopener noreferrer nofollow"
+                >Children's Privacy Policy</a
```
Comment (lukasz-rdzanek) [inline pages/housePrideQuiz/hufflepuff/hufflepuffNewt.html:347]: This </a> close tag should go to next line

File: pages/housePrideQuiz/hufflepuff/hufflepuffNewt.html [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+              >
+            </li>
+            <li class="mobileFooter__item">
+              <a
+                href="#"
+                class="mobileFooter__link"
+                target="_blank"
+                rel="noopener noreferrer nofollow"
+                >Legals</a
```
Comment (lukasz-rdzanek) [inline pages/housePrideQuiz/hufflepuff/hufflepuffNewt.html:356]: Same as above

**Review by lukasz-rdzanek — CHANGES_REQUESTED** [review]
Fix issues in comments

File: pages/housePrideQuiz/hufflepuff/hufflepuffNewt.html (hunk 1 above)
Comment (Ervige) [reply pages/housePrideQuiz/hufflepuff/hufflepuffNewt.html:347]: @lukasz-rdzanek fixed

File: pages/housePrideQuiz/hufflepuff/hufflepuffNewt.html (hunk 2 above)
Comment (Ervige) [reply pages/housePrideQuiz/hufflepuff/hufflepuffNewt.html:356]: @lukasz-rdzanek fixed

**Review by lukasz-rdzanek — APPROVED** [review]

**Pull request merged**